changelogs/fragments/62188-VMware-Guest-Support-latest-version-while-upgrading-VM-hardware.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- VMWare Support latest version while upgrading Virtual Machine hardware. It is VMWare feature and latest param will upgrade to latest hardware version Issue #56273.
```

Comment:
I still don't like it :-), I suggest:

- vmware_guest - Add ability to upgrade the guest hardware version to ``latest``.

Reply:
Thanks @goneri 
Done the changes. Please review